### PR TITLE
Anti-RightClick: outdoornews.co.kr

### PR DESCRIPTION
**Pull request by ghost**
### URL
www.outdoornews.co.kr/news/articleView.html?idxno=32695

### CODE REVIEW

File: AnnoyancesFilter/sections/tweaks.txt [hunk 1]
```diff
@@ -1337,6 +1337,9 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.oncontextmenu', 'null')
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.onselectstart', 'null')
+outdoornews.co.kr#%#//scriptlet('set-constant', 'document.body.ondragstart', 'null')
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1342]: What are these rules for?

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -1368,7 +1371,7 @@ codedosa.com,newslibrary.naver.com,insight.co.kr,jootc.com,gamefinity.id,blog.da
 insight.co.kr,jootc.com,blog.daum.net,comic.naver.com,cafe.naver.com#%#//scriptlet('set-constant', 'document.onselectstart', 'null')
 codedosa.com,gradium.co.kr,insight.co.kr,jootc.com,gamefinity.id,blog.daum.net,comic.naver.com,cafe.naver.com#%#//scriptlet('set-constant', 'document.ondragstart', 'null')
 comic.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'img[id^="content_image_"]')
-egloos.com,novel.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'body')
+outdoornews.co.kr,egloos.com,novel.naver.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu', 'body')
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1374]: It seems this one is enough.

Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt:1371]: Okay. https://github.com/AdguardTeam/AdguardFilters/pull/90512/commits/ce1c4e69e929ab00ec350ca5cdaaeac3fe7800b8

File: AnnoyancesFilter/sections/tweaks.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/90512/commits/ce1c4e69e929ab00ec350ca5cdaaeac3fe7800b8

**Review by zloyden — APPROVED** [review]

**Pull request merged**